.github/workflows/release-dev-builder-images.yaml

```diff
@@ -28,58 +24,108 @@ jobs:
     name: Release dev builder images
     if: ${{ inputs.release_dev_builder_ubuntu_image || inputs.release_dev_builder_centos_image || inputs.release_dev_builder_android_image }} # Only manually trigger this job.
     runs-on: ubuntu-20.04-16-cores
+    outputs:
+      version: ${{ steps.set-version.outputs.version }}
     steps:
       - name: Checkout
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
+      - name: Configure build image version
+        id: set-version
+        shell: bash
+        run: |
+          commitShortSHA=`echo ${GITHUB_SHA} | cut -c1-8`
```

Comment:
suggestion: We can add the rust tool chain version, for example:

```
RUST_TOOLCHAIN_VERSION=$(cat rust_toolchain.toml | grep  -Eo '[0-9]{4}-[0-9]{2}-[0-9]{2}')
```

The final version tag can be: `${RUST_TOOLCHAIN}-${BUILD_TIME}-${SHORT_SHA}`

Reply:
For example:

```
2024-06-06-20240826162820-46a8ffa
```